pyne/mcnp.py

```diff
@@ -2123,11 +2119,11 @@ def read_xyze_bounds(self, f):
         while line.strip() != 'Tally bin boundaries:':
             line = f.readline()
 
-        x_bounds = [float(x) for x in f.readline().split()[2:]]
-        y_bounds = [float(x) for x in f.readline().split()[2:]]
-        z_bounds = [float(x) for x in f.readline().split()[2:]]
+        x_bounds = tuple([float(x) for x in f.readline().split()[2:]])
```

Comment:
Small change for readability. Applies to `y/z/e_bounds` too.

```suggestion
        x_bounds = tuple(float(x) for x in f.readline().split()[2:])
```